packages/lesswrong/components/common/HashLink.tsx

```diff
@@ -84,7 +85,14 @@ export function genericHashLink(props, As) {
 }
 
 export function HashLink(props) {
-  return genericHashLink(props, Link);
+  // React router links don't handle external URLs, so use a
+  // normal HTML a tag if the URL is external
+  const externalLink = props.to.slice(0,8) === 'https://'  
+  const Element = externalLink ? 
+    (p) => React.createElement('a', {...p, href: p.to})
```

Comment:
`React.createElement` isn't really idiomatic. I'd rewrite this in plain JSX:

```js
({to, ...rest}) => <a href={to} {...rest} />
```

Reply:
You might also want to consider adding `target='_blank'` to external links to open them in a new tab (not sure). If you do, [make sure you also add `rel="noopener"`](https://web.dev/external-anchors-use-rel-noopener/)